tree/checkMirrorImage.js

```diff
@@ -0,0 +1,20 @@
+const BinaryTree = require("./BinaryTree");
+
+let checkMirrorImage = (tree1, tree2) => {
+    let isMirrorImage = true;
```

Comment:
why do you want to set this to true by default?

Reply:
i have set is as true and checking and changing it to false when a mismatch is found. if no mismatch found, it will remain true and returned.